docs/components/modeler/desktop-modeler/element-templates/additional-resources.md

```diff
@@ -3,7 +3,7 @@ id: additional-resources
 title: Additional resources
 ---
 
-Use the [Camunda Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/) and retrieve templates from a Cawemo catalog project.
+Use the [Camunda Platform 8 Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/) and retrieve templates from a Cawemo catalog project.
```

Comment:
The Plugin will not be renamed

Reply:
And moreover, the plugin is for Camunda Platform 7, not 8.

Reply:
So this is the "Cloud Connect plugin"?

Reply:
Do you want me to be more clear that this is for C7? Usually, we badge the page or section of "Camunda Platform 7 only".